src/util/LinkTypes.ts

```diff
@@ -0,0 +1,2 @@
+export const LINK_TYPE_LDP_C = '<http://www.w3.org/ns/ldp#Container>; rel"type"';
```

Comment:
Isn't that a bit annoying? Because that `rel="type"` allows us to talk about link types, otherwise they're just URIs like in the new Prefixes file (without the `#Container`). So then I have to check in the InteractionController if `<${LINK_TYPE_LDP_C}>; rel="type"` is in there, which makes it less clear over there.

Reply:
I didn't check the exact code, but I would expect the InteractionController to check `metadata.linkRel?.type`, which should contain the value of `LINK_TYPE_LDP_C` (which is the responsibility of a parser somewhere earlier to parse that).

Reply:
It does, but it is important that in the Link header also the `rel="type"` is given, or do you mean that this will be checked in another place and then only `'http://www.w3.org/ns/ldp#Container'` will be added to the Set `metadata.linkRel?.type`?

Reply:
The second. At some point another class would convert all those metadata elements `<${metadata.linkRel.type}>; rel=type"`, and similarly for other links. This is a syntax thing so the ResourceStore shouldn't bother with things like that. It should only care about the unique identifier that corresponds to a container or resource.